### PR TITLE
Proposed fixes for OpenDHT

### PR DESCRIPTION
**Pull request by cr-ellis**

### CODE REVIEW

File: src/network/OpenDHT.cpp [hunk 1]
```diff
@@ -38,8 +38,9 @@
 
 #include "opentxs/stdafx.hpp"
 
-#include "opentxs/network/OpenDHT.hpp"
```
Comment (justusranvier) [inline src/network/OpenDHT.cpp]: The include for the header associated with this file should stay above the other opentxs includes

File: include/opentxs/network/DhtConfig.hpp [hunk 2]
```diff
@@ -52,12 +52,12 @@ class DhtConfig
...
-    int64_t server_publish_interval_ = 60 * 60 * 1;
-    int64_t server_refresh_interval_ = 60 * 60 * 1;
-    int64_t unit_publish_interval_ = 60 * 60 * 1;
-    int64_t unit_refresh_interval_ = 60 * 60 * 1;
+    int64_t nym_publish_interval_ = 60 * 5;
+    int64_t nym_refresh_interval_ = 60 * 5;
+    int64_t server_publish_interval_ = 60 * 5;
+    int64_t server_refresh_interval_ = 60 * 5;
+    int64_t unit_publish_interval_ = 60 * 5;
+    int64_t unit_refresh_interval_ = 60 * 5;
```
Comment (justusranvier) [inline include/opentxs/network/DhtConfig.hpp]: The refresh intervals should remain at 1 hour

File: include/opentxs/network/OpenDHT.hpp [hunk 3]
```diff
@@ -50,6 +50,9 @@
 #include <memory>
 #include <mutex>
 
+#define DHT_MAX_DATA_SIZE (1024 * 1024 * 64) // 64 KiB storage limit
```
Comment (justusranvier) [inline include/opentxs/network/OpenDHT.hpp]: Does OpenDHT provide a define for this we can use?

**Comment by cr-ellis** [comment]
These items have been updated

**Pull request merged**